bots/models.py

```diff
@@ -49,8 +49,7 @@ def choices_for_pr(
         if not pr.workspace or pr.workspace.is_personal:
             return {cls.UNLISTED, cls.PUBLIC, PublishedRunVisibility(pr.visibility)}
         else:
-            # TODO: Add cls.PUBLIC when team-handles are added
-            return {cls.UNLISTED, cls.INTERNAL, PublishedRunVisibility(pr.visibility)}
+            return [cls.UNLISTED, cls.INTERNAL, cls.PUBLIC]
```

Comment:
might want to `list(PublishedRunVisibility)` here?

Reply:
The default order is different from what we want on the UI (unlisted -> internal -> public)